cypress/fixtures/elements.json

```diff
@@ -29,9 +30,9 @@
     },
     "modalElements": {
         "modalHeaderText": ".modal__header h3",
-        "modalProjectNameField": ".modal__body #input_add-project-name",
+        "modalProjectNameField": ".modal__body [id$=\"project-name\"]",
         "modalSaveButton": ".modal__footer .btn--primary",
```

Comment:
Would you prefer it if we start putting ID's on these for you? Alternatively, don't be afraid to go in and add them yourself.

Reply:
If you could that would make my life much easier and anyone else's when it comes to maintaining this in future if I'm not helping out anymore :)
But for now, we can add that for improvements for the next phase as for now it's all okay